recipes-bsp/tegra-binaries/tegra-configs_34.1.0.bb

```diff
@@ -48,11 +48,13 @@ do_install:append:tegra194() {
 
 do_install:append:tegra234() {
     install -m 0644 ${S}/etc/X11/xorg.conf ${D}${sysconfdir}/X11/xorg.conf
+    install -d ${D}${sysconfdir}/X11/xorg.conf.d
+    install -m 0644 ${S}/etc/X11/xorg.conf.d/tegra-drm-outputclass.conf ${D}${sysconfdir}/X11/xorg.conf.d
 }
 
 PACKAGES = "${PN}-udev ${PN}-omx-tegra ${PN}-xorg ${PN}-nvstartup ${PN}-container-csv"
 FILES:${PN}-udev = "${sysconfdir}/udev/rules.d"
-FILES:${PN}-xorg = "${sysconfdir}/X11"
+FILES:${PN}-xorg = "${sysconfdir}/X11 ${sysconfdir}/X11/xorg.conf.d"
```

Comment:
This addition wasn't strictly necessary, but that's OK

Reply:
I will remove it